### PR TITLE
new readme and docker install script

### PR DESCRIPTION
**Pull request by c-brenn**
General documentation changes as per feedback #92 

Opening for discussion, will update changelog etc now.

### CODE REVIEW

File: BUILDING_MANUALLY.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+You should revert these settings when leaving the TCD network by running
+
+```bash
+$ sudo rm /etc/docker/daemon.json
+$ sudo service docker restart
+```
+
+1) Clone the Repo
```
Comment (houli) [inline BUILDING_MANUALLY.md]: Markdown numbered lists `1.`

Comment (c-brenn) [reply BUILDING_MANUALLY.md]: fixing now - figuring out source blocks in lists

Comment (c-brenn) [reply BUILDING_MANUALLY.md]: after much trial and error, this is fixed and code indented properly

**Comment by c-brenn** [comment]
closes #95 and #92

**Comment by 22a** [comment]
Looking great chaps 🍾

**Comment by c-brenn** [comment]
help me - pls approve @22a

**Review by 22a — APPROVED** [review]
Astonishing work 🤙

**Pull request merged**